x/wasm/types/params.go

```diff
@@ -14,28 +14,34 @@ import (
 )
 
 const (
-	// DefaultParamspace for params keeper
-	DefaultParamspace = ModuleName
-	// DefaultMaxWasmCodeSize limit max bytes read to prevent gzip bombs
-	DefaultMaxWasmCodeSize = 600 * 1024 * 2
-	// GasMultiplier is how many cosmwasm gas points = 1 sdk gas point
-	// SDK reference costs is defined in x/store/types/gas.go KVGasConfig()
-	// https://github.com/line/lbm-sdk/blob/main/store/types/gas.go
-	// A write at ~3000 gas and ~200us = 10 gas per us (microsecond) cpu/io
-	// Rough timing have 88k gas at 90us, which is equal to 1k sdk gas... (one read)
+	// DefaultGasMultiplier is how many CosmWasm gas points = 1 Cosmos SDK gas point.
+	//
+	// CosmWasm gas strategy is documented in https://https://github.com/line/cosmwasm/blob/v1.0.0-0.6.0/docs/GAS.md.
+	// LBM SDK reference costs can be found here: https://github.com/line/lbm-sdk/blob/main/store/types/gas.go#L199-L209.
+	//
+	// The original multiplier of 100 up to CosmWasm 0.16 was based on
+	//     "A write at ~3000 gas and ~200us = 10 gas per us (microsecond) cpu/io
+	//     Rough timing have 88k gas at 90us, which is equal to 1k sdk gas... (one read)"
+	// as well as manual Wasmer benchmarks from 2019. This was then multiplied by 150_000
+	// in the 0.16 -> 1.0 upgrade (https://github.com/CosmWasm/cosmwasm/pull/1120).
+	//
+	// The multiplier deserves more reproducible benchmarking and a strategy that allows easy adjustments.
+	// This is tracked in https://github.com/CosmWasm/wasmd/issues/566 and https://github.com/CosmWasm/wasmd/issues/631.
+	// Gas adjustments are consensus breaking but may happen in any release marked as consensus breaking.
+	// Do not make assumptions on how much gas an operation will consume in places that are hard to adjust,
+	// such as hardcoding them in contracts.
 	//
 	// Please not that all gas prices returned to the wasmer engine should have this multiplied
-	DefaultGasMultiplier uint64 = 100
+	DefaultGasMultiplier uint64 = 140_000_000
```

Comment:
Is it no problem? 
The previous value have been changed by us and the difference is too large.

Reply:
Yes, These are the values that the CosmWasm team has considered.
Even if there is a possibility that our adjustments may change it in the future, we would like to keep it as it is at the time of migration.

FYI↓
https://github.com/CosmWasm/wasmd/pull/634#issuecomment-938055852
https://github.com/CosmWasm/wasmd/issues/631

Reply:
OK, thank you. But I think we also need to check if it is proper `DefaultGasMultiplier` or not later.